packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -272,7 +272,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     let { items } = this.state;
 
     if (items.length && index! >= 0) {
-      let newEl: HTMLElement = this.root.querySelectorAll('[data-selection-index]')[Math.min(index!, items.length - 1)] as HTMLElement;
+      let newEl: HTMLElement = this.root.querySelectorAll('[data-selection-index]') [Math.min(index!, items.length - 1)] as HTMLElement;
```

Comment:
formatting changes doen't seem right

Reply:
That was automatic. I didn't want to do it. I'll change it.